### ISSUE TITLE
BarChart: Slider with tickmarks does not have correct height 

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#41552]
The slider field option does not fully work with the step marks, they extend beyond it's height making the spacing to the next option to small and different to the rest of the options. 

![Screenshot from 2021-11-10 16-00-46](https://user-images.githubusercontent.com/10999/141137199-bf7f1ddb-c4e2-4d8d-84cd-d3292eb12361.png)

**Comment by leeoniya** [comment]
yep, mentioned this to @oscarkilhed a few days ago. i think the plan was to roll it into https://github.com/grafana/grafana/pull/41362 ?
